index.js

```diff
@@ -1310,6 +1311,28 @@
   }
   S.fromMaybe = def('fromMaybe', {}, [a, $Maybe(a), a], fromMaybe);
 
+  //# fromMaybe_ :: (() -> a) -> Maybe a -> a
+  //.
+  //. Variant of [`fromMaybe`](#fromMaybe) which takes a thunk so the default
+  //. value is only computed if required.
+  //.
+  //. ```javascript
+  //. > global.fib = function fib(n) {
+  //. .   return n <= 1 ? n : fib(n - 2) + fib(n - 1);
+  //. . }
+  //. fib
```

Comment:
Thanks for highlighting this, Aldwin. We are indeed working around a limitation of the tool. I just opened davidchambers/doctest#73. At some point I will address that issue, publish a new version of doctest, and update these `fib` definitions. This needn't all happen before this pull request is merged, though. :)